### PR TITLE
Archival Storage: Fix pagination

### PR DESCRIPTION
**Pull request by Hwesta**
Fix pagination of AIPs and AIP files using Django's Paginator.  To avoid fetching all results (to get correct pagination information), use LazyPagedSequence that fetches from ElasticSearch as needed.

Other cleanup:
- Return deep copy of MATCH_ALL_QUERY so changes don't persist across requests
- Update search_raw_wrapper to search_all_results and rename parameters to be a drop-in replacement for conn.search.  Since this didn't end up being needed I may remove it
- Use _pager.html for paging in archival storage, instead of a custom one
- Submit archival storage search on Enter
- Remove unused templates

### CODE REVIEW

File: src/dashboard/src/components/archival_storage/views.py [hunk 1]
```diff
@@ -538,7 +555,7 @@ def list_display(request):
 
             try:
                 size = '{0:.2f} MB'.format(float(aip['size']))
-            except TypeError, ValueError:
+            except (TypeError, ValueError):
```
Comment (mistydemeo) [inline src/dashboard/src/components/archival_storage/views.py:558]: 😱

**Comment by mistydemeo** [comment]
:+1: 

Should I go ahead and publish LazyPagedSequence to pypi so we can use it here and in the pair matcher?

**Comment by Hwesta** [comment]
Yes, LazyPagedSequence being in PyPI would be great.

**Comment by mistydemeo** [comment]
:+1:

**Pull request merged**